fe/fe-core/src/main/java/com/starrocks/lake/compaction/PartitionStatistics.java

```diff
@@ -32,6 +32,25 @@ public class PartitionStatistics {
     @SerializedName(value = "compactionScore")
     private Quantiles compactionScore;
 
+    // default priority is -1, manual compaction will have priority value 1
+    @SerializedName(value = "priority")
+    private volatile CompactionPriority priority = CompactionPriority.DEFAULT;
+
+    public enum CompactionPriority {
+        DEFAULT(-1),
```

Comment:
Is there any special reason that you choosed `-1` as the default value instead of `0`?

Reply:
better use 0 as default